### ISSUE TITLE
divide the dataset

### ISSUE THREAD

**Issue by ChouCHou-y** [ghimireadarsh/Semantic-Segmentation-using-Unet#1]
Hello, I would like to ask, how do you divide the dataset, I mean training set, validation set, test se

**Comment by ghimireadarsh** [comment]
Hi, Out of 2400 images, I took - the first 2000 images for training, the next 200 for validation and last 200 for the test set. It's in the [notebook](https://github.com/ghimireadarsh/Semantic-Segmentation-using-Unet/blob/main/notebook.ipynb) on cell numbers 8, 9, 10, 11.

**Comment by ChouCHou-y** [comment]
> Hi, Out of 2400 images, I took - the first 2000 images for training, the next 200 for validation and last 200 for the test set. It's in the [notebook](https://github.com/ghimireadarsh/Semantic-Segmentation-using-Unet/blob/main/notebook.ipynb) on cell numbers 8, 9, 10, 11.

okay.I got it.Thanks a million
